ipaserver/plugins/baseuser.py

```diff
@@ -376,6 +387,14 @@ class baseuser(LDAPObject):
             cli_name='radius_username',
             label=_('RADIUS proxy username'),
         ),
+        Str('ipaidpconfiglink?',
+            cli_name='idp',
+            label=_('External IdP configuration'),
+            ),
+        Str('ipaidpsub?',
+            cli_name='idp_user',
+            label=_('External IdP user name'),
```

Comment:
Maybe it would be better to call it more generic 'External IdP user identifier' and the option 'idp_user_id', since even with the templates it is typically the email and not the name. It might be even worth to make this better with the option of idp-{add|mod} and replace '--subject' with '--user-id-attribute' so that 'user-id' is used in both places.

Reply:
Thanks. I changed it to `--idp-user-id` and updated doc/label strings to get it better represented.